resources/views/feedbackboards/include/feedback.blade.php

```diff
@@ -44,37 +45,39 @@
     </div>
 
     <div class="card-body">
-            {!! $feedback->feedback !!}
+        {!! $feedback->feedback !!}
```

Comment:
I know that this was already here, but are we sure that this feedback is safe?

Reply:
No and also we should not want to render html in the feedback anyways. The feedback should only be plain text in my opinion.

Reply:
Yeah I agree, fixed now